code/datums/quirks/negative.dm

```diff
@@ -803,3 +803,20 @@
 		SEND_SIGNAL(quirk_holder, COMSIG_ADD_MOOD_EVENT, "bad_touch", /datum/mood_event/very_bad_touch)
 	else
 		SEND_SIGNAL(quirk_holder, COMSIG_ADD_MOOD_EVENT, "bad_touch", /datum/mood_event/bad_touch)
+
+/datum/quirk/monochromatic
```

Comment:
i dont know how quirk compatibility works, but this will be incompatible with blind right?

Reply:
Yea, I'm making a commit right now. Already noticed, but good eye.